target/linux/ramips/dts/mt7628an_tplink_tl-wr902ac-v4.dts

```diff
@@ -0,0 +1,112 @@
+#include "mt7628an_tplink_8m.dtsi"
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "tplink,tl-wr902ac-v4", "mediatek,mt7628an-soc";
+	model = "TP-Link TL-WR902AC v4";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		button-reset {
+			label = "reset";
+			gpios = <&gpio 38 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		button-sw1 {
+			label = "sw1";
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+			linux,code = <BTN_0>;
+		};
+
+		button-sw2 {
+			label = "sw2";
+			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
+			linux,code = <BTN_1>;
+		};
+
+		button-wps {
+			label = "wps";
+			gpios = <&gpio 41 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_lan {
+			label ="green:lan";
+            color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_LAN;
+			gpios = <&gpio 5 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_power: power {
+			label = "green:power";
```

Comment:
not needed (auto-created from `color` and `function`).

But this needs #13837 to work with the aliases above.

Reply:
> not needed (auto-created from `color` and `function`).
> 
> But this needs #13837 to work with the aliases above.

So, in fact "label" is needed here :)

Should I wait for https://github.com/openwrt/openwrt/pull/13837  or left 'label' for power led? (Rest leds seems to work without label)

Reply:
renamed to led-x, I left label for now

Reply:
I believe that the other PR will be merged soon; you could just wait and see when it does and then fix.

Reply:
Please create a follow up PR after https://github.com/openwrt/openwrt/pull/13837  was merged

Reply:
It has just been merged, so labels may be dropped now.